### DISCUSSION TITLE
Do I need to rebuild and deploy after renewing my licence?

### DISCUSSION THREAD

**Discussion by OldFlexDev** [airsdk/Adobe-Runtime-Support#3183] [General]
I have an app in production and my Harman AIR license is due to expire soon. After I renew, do I need to rebuild and deploy the app or will it just continue to work? The renewal page seems to suggest that this isn't necessary but I don't want to get a nasty surprise on renewal day.

**Comment by ajwfrost** [comment]
With AIR SDK, if you create an application package with a licensed version then it will continue to work. The license is required at the point of packaging it up.. no nasty surprises!
